x-pack/test/security_solution_cypress/cypress/screens/breadcrumbs.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const RULES_TABLE_BREADCRUMB =
+  '[data-test-subj~="breadcrumb"][title="Detection rules (SIEM)"]';
```

Comment:
Is there a way to make breadcrumbs' CSS selectors more stable? E.g. assign a custom `data-test-subj` attribute to a specific breadcrumb, so this selector would become something like this:

```ts
export const RULES_TABLE_BREADCRUMB = '[data-test-subj="breadcrumbRuleManagementPage"]';
```

Reply:
Unfortunately we don't have control of `data-test-subj` for breadcrumbs. On top of that additional `data-test-subj` are added on different levels for example [here](https://github.com/elastic/kibana/blob/main/packages/core/chrome/core-chrome-browser-internal/src/project_navigation/project_navigation_service.ts#L156-L158) and [here](https://github.com/elastic/kibana/blob/main/packages/core/chrome/core-chrome-browser-internal/src/ui/header/header_breadcrumbs.tsx#L35-L40). Obviously it can be refactored to support custom `data-test-subj` values but it's not a simple change. I'd investigate this possibility separately.